unit_tests/src/test_rc_data_obj.cpp

```diff
@@ -94,27 +92,27 @@ TEST_CASE("open,read,write,close")
 
             // ensure all system metadata were updated properly
             const auto [replica_info, replica_lm] = replica::make_replica_proxy(comm, target_object, 0);
-            CHECK(replica_info.mtime() == replica_info.ctime());
+            //CHECK(replica_info.mtime() == replica_info.ctime());
```

Comment:
torch if dead code.

Reply:
Actually, not dead code. I've been restoring these checks for this test as they are valid.